packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts

```diff
@@ -68,12 +68,15 @@ export function createRichTextV2Plugin<T extends ControlDefinition>({
   return { control, withPlugin, onKeyDown, renderElement, renderLeaf }
 }
 
-type RichTextV2Config = {
-  plugins?: RichTextV2Plugin[]
+export type RichTextV2Config = {
   mode?: RichTextV2Mode
+  plugins?: RichTextV2Plugin[]
+  defaultValue?: Descendant[]
 }
 
-type ExternalRichTextV2Config = Omit<RichTextV2Config, 'plugins'>
+type ExternalRichTextV2Config = Omit<RichTextV2Config, 'plugins' | 'defaultValue'> & {
+  unstable_defaultValue?: Descendant[]
```

Comment:
I am exposing the `unstable_defaultValue` so that this component can be used in the `Text` component